app/services/submission/MessageIdentificationService.scala

```diff
@@ -14,13 +14,13 @@
  * limitations under the License.
  */
 
-package pages.sections
+package services.submission
 
-import play.api.libs.json.{JsObject, JsPath}
+import javax.inject.Inject
+import scala.util.Random
 
-case object NavigationDataSection extends Section[JsObject] {
+class MessageIdentificationService @Inject() {
 
-  override def path: JsPath = JsPath \ toString
+  def randomIdentifier: String = Random.alphanumeric.take(35).mkString
```

Comment:
is the alphanumeric thing always greater than 35?

Reply:
@joecefnhedog yeah it's an infinite lazy list